crates/kitsune_p2p/kitsune_p2p/src/fixt.rs

```diff
@@ -17,7 +17,7 @@ fixturator!(
     curve Empty vec![];
     curve Unpredictable {
         let mut rng = ::fixt::rng();
-        let vec_len = rng.gen_range(0..5);
+        let vec_len = rng.gen_range(1..3);
```

Comment:
Yep : ) - Mostly it just can't be zero, because that has a special meaning, indicating the agent is not active. If I implement the multi-signal server stuff we *may* have two in here sometimes, the global holochain one, and a local LAN one. Three should be exceedingly rare.